env_linux.sh.template

```diff
@@ -1,14 +1,25 @@
 #!/bin/bash
-export HOST_NAME=<name>
-export HOST_DOMAIN=<domain>
-export NUM_HOSTS=<num_hosts>
+
+# THE USER SHOULD REPLACE ALL variables which are set to <...>
+ 
+export CLUSTER_NAME=<name>       # How you want your hosts be called
+export HOST_DOMAIN=<domain>   # How you want your domain to be 
+export NUM_HOSTS=<num_hosts>  # How many host you would like to have
+
+# Define home for Vagrant. It is where the plugins will be installed.
+# Use double back slash to express a window path
+# Default value in Windows is the following:
+# C:\\HashiCorp\\Vagrant\\home
+export VAGRANT_HOME=<vagrant_home>
+
+
+
+# DO NOT EDIT FROM HERE ON
 
 #Vagrant will use 80% of the total main memory
 total_memory=$(grep MemTotal /proc/meminfo | awk '{print $2}')
 export MEM_SIZE=$(calc $total_memory*0.8/$NUM_HOSTS/1024 | awk '{printf("%d",$1 + 0.5)}')
 echo "Memory per node: $MEM_SIZE MB"
 
-export KEYS_LOCATION=/tmp/$HOST_NAME
-#Use double back slash to express window path
-#Default value in Windows is C:\\HashiCorp\\Vagrant\\home
-export VAGRANT_HOME=<vagrant_home>
+export KEYS_LOCATION=/tmp/$CLUSTER_NAME
```

Comment:
When you reboot your machine, the /tmp will be wiped, causing you to loose the private key to login to the cluster.

Can we change it to a subdir in the current working directory?